Makefile

```diff
@@ -52,14 +52,19 @@ uv: ## Install UV
 ruff: ## Install Ruff
 	@uvx ruff --help &> /dev/null || uv tool install ruff
 
-
 .PHONY: verify
 verify: uv ruff  ## install all required tools
 	@cd $(PY_DIR) && uv lock --check
 	@cd $(PY_DIR) && uvx ruff check --show-fixes
 
-.PHONY: test-python
-test-python: ## Run Python unit test.
-	uv install --dev
-	PYTHONPATH=$(PROJECT_DIR) pytest ./python/kubeflow
-
+ # make test-unit will produce html coverage by default. Run with `make test-unit report=xml` to produce xml report.
+.PHONY: test-unit
+test-unit:
```

Comment:
Maybe we should keep instruction name `test-python` for consistency with Trainer ?
https://github.com/kubeflow/trainer/blob/master/Makefile#L162